### PR TITLE
add twirling_critic

### PR DESCRIPTION
**Pull request by artofnothingness**

### CODE REVIEW

**Comment by artofnothingness** [comment]
This one too temporarily merge. I guess i need to make changes in one branch, so it would be easier to merge, sync them

File: critics.xml [hunk 1]
```diff
@@ -29,6 +29,10 @@
       <description></description>
     </class>
 
+    <class type="mppi::critics::TwirlingCritic" base_class_type="mppi::critics::CriticFunction">
+      <description></description>
```
Comment (SteveMacenski) [inline critics.xml:33]: Add description: mppi critic for preventing twirling behavior when using omnidirectional models

**Review by SteveMacenski — COMMENTED** [review]
LGTM in general, just that one line

File: critics.xml (hunk 1 above)
Comment (artofnothingness) [reply critics.xml:33]: Done in new PR

**Pull request merged**